src/lib/components/ratio/ColorIssues.svelte

```diff
@@ -81,6 +71,8 @@
     grid-auto-rows: auto;
     grid-template-columns: 1fr;
     margin-block-start: var(--gutter);
+    max-width: 85ch;
```

Comment:
we could go max-inline-size here as we are using logical properties elsewhere.

Reply:
good catch. fixed in https://github.com/oddbird/oddcontrast/pull/223/commits/1e6c2902ab2e7d0f9af9d9930fd4639edb6e6c3c